docs/docs-graphql/schema.graphqls

```diff
@@ -3742,6 +3818,22 @@ The `ScalarTypeDefID` scalar type represents an identifier for an object of type
 """
 scalar ScalarTypeDefID
 
+type SearchResult {
+  filePath: String!
```

Comment:
This feels a little odd to me - definitely makes sense in the context of `Directory.search`, but `File.search` doesn't really have a `path`, it only really has a `name`.

No real other suggestions, so SGTM :tada:

Reply:
Well - to me the `name` is still a path, it's just `foo.txt` in an otherwise empty directory. I don't think it's _wrong_ for this to be here, or for it to be populated in file search results.

I did deliberate for a bit on adding a `File.path` attribute that would be e.g. `core/file.go` in `coreDir.File("file.go")` but backed out as it seemed like an information leak.